packaging/ubuntu-14.04/snapd.postrm

```diff
@@ -39,6 +39,9 @@ if [ "$1" = "purge" ]; then
         rev=$(grep 'Where=/snap/' "/etc/systemd/system/$unit"|cut -f4 -d/)
         if [ -n "$snap" ]; then
             echo "Removing snap $snap"
+            # aliases
+            find /snap/bin -lname "$snap"|xargs rm -f
+            find /snap/bin -lname "$snap.*"|xargs rm -f
```

Comment:
this will fail if `find` finds nothing (GNU `xargs` has `-r` for this). It's also bad practice to not use `-print0` / `-0` for `xargs`. Lastly it might be simpler, in this case, to do `find /snap/bin -lname "..." -delete`.